lib/log-group.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-console */
+
+/**
+ * A simple helper to manage the print of a set of logs
+ */
+class LogGroup {
+  /**
+   * @param {object} input
+   * @param {string} input.title
+   * @param {boolean} input.isPrimary
+   */
+  constructor({title, isPrimary}) {
```

Comment:
Using a default parameter makes this a little nicer, since otherwise the object destructuring will fail when you forget to pass in an empty object:

```js
constructor({title, isPrimary} = {}) {
}
```